test/integration/client-side-encryption/client_side_encryption.prose.18.azure_kms_mock_server.test.ts

```diff
@@ -119,7 +119,7 @@ context('Azure KMS Mock Server Tests', function () {
     it('returns an error after the request times out', async () => {
       const error = await fetchAzureKMSToken(new KMSRequestOptions('slow')).catch(e => e);
 
-      expect(error).to.be.instanceof(MongoCryptAzureKMSRequestError);
+      expect(error).to.be.instanceof(MongoNetworkTimeoutError);
```

Comment:
This is an error that could reach users I believe. What's the reason for this changing? Shouldn't this remain the same API?

Reply:
I've fixed to throw the current error now.